cmd/list/dnsdomains/cmd.go

```diff
@@ -89,18 +100,32 @@ func run(_ *cobra.Command, _ []string) {
 	// Create the writer that will be used to print the tabulated results:
 	writer := tabwriter.NewWriter(os.Stdout, 0, 4, 2, ' ', 0)
 
-	fmt.Fprintf(writer, "ID\tCLUSTER ID\tRESERVED TIME\tUSER DEFINED\n")
+	fmt.Fprintf(writer, "ID\tCLUSTER ID\tRESERVED TIME\tUSER DEFINED\tARCHITECTURE\n")
 	for _, dnsdomain := range dnsDomains {
-		userDefind := "No"
+		userDefined := "No"
 		if dnsdomain.UserDefined() {
-			userDefind = "Yes"
+			userDefined = "Yes"
 		}
-		fmt.Fprintf(writer, "%s\t%s\t%s\t%s\n",
+		fmt.Fprintf(writer, "%s\t%s\t%s\t%s\t%s\n",
 			dnsdomain.ID(),
 			dnsdomain.Cluster().ID(),
 			dnsdomain.ReservedAtTimestamp().Format(time.RFC3339),
-			userDefind,
+			userDefined,
+			dnsdomain.ClusterArch(),
 		)
 	}
 	writer.Flush()
 }
+
+func filterByBaseDomain(domains []*v1.DNSDomain, isHostedCp bool) []*v1.DNSDomain {
```

Comment:
Would rather move the bool outside of the function instead if you agree
https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3